boot/seal.go

```diff
@@ -141,20 +139,43 @@ func fallbackKeySealRequests(key, saveKey secboot.EncryptionKey) []secboot.SealK
 	}
 }
 
+type fdeSetupHookResult struct {
```

Comment:
Maybe we should include v2 in the name?

Reply:
I was thinking about this but would love to have as few references to v2 if possible, v2 is the "normal" case (until v3 ;)

Reply:
this actually should probably live in devicestate/fde (which in itself maybe should be moved, but that can happen later) ?

Reply:
I moved that over now

Reply:
Well, I move it next to FDESetupHookParams because right now this is used in "boot" and I don't think we want to import "overlord/devicemanager/fde" from there - however we could move the entire devicemanager/fde into boot/fde or secboot/fde - I don't think we do anything overlord-ish there. Happy to do this here or in a followup if desired.